packages/core/src/test/applicationcomposer/messageHandlers/initMessageHandler.test.ts

```diff
@@ -7,13 +7,11 @@ import assert from 'assert'
 import sinon from 'sinon'
 import { initMessageHandler } from '../../../applicationcomposer/messageHandlers/initMessageHandler'
 import { createTemplate, createWebviewContext } from '../utils'
+import { skipIfVscodeBelow } from '../../testUtil'
 
 describe('initMessageHandler', function () {
-    afterEach(() => {
-        sinon.restore()
-    })
```

Comment:
does this risk new tests being added which forget to restore stuff?

Reply:
Since we are mocking within the test block (not in a before or beforeEach) I thought we should restore it within the same test block to keep to contained. But I don't see any problem with restoring outside since there's no danger in an extra `restore` call.